debian/clearwater-socket-factory.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Enables processes to establish connections using a different network namespace
+
+[Service]
+ExecStart=/usr/share/clearwater/bin/clearwater-socket-factory-wrapper
+Restart=on-failure
+LimitCORE=infinity
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Is there any way to pace the respawn like we do with upstart? If not should we put the sleep in the wrapper?